src/legacy/ui/public/field_editor/components/field_format_editor/editors/url/url.js

```diff
@@ -113,6 +120,44 @@ export class UrlFormatEditor extends DefaultFormatEditor {
     });
   }
 
+  renderWidthHeightParameters = () => {
+    const { error } = this.state;
+    const width = this.sanitizeNumericValue(this.props.formatParams.width);
+    const height = this.sanitizeNumericValue(this.props.formatParams.height);
+    return (
+      <Fragment>
+        <EuiFormRow
+          label={<FormattedMessage id="common.ui.fieldEditor.url.widthLabel" defaultMessage="Width" />}
+          isInvalid={!!error}
+          error={error}
+        >
+          <EuiFieldNumber
+            data-test-subj="urlEditorWidth"
+            placeholder={128}
```

Comment:
I think we should leave the `placeholder` out (or change it to something different). Imho the `128` placeholder suggests that 128 would be our default value (at least that's how we use it in couple other places), which is not true here. Do we need any placeholder at all?

Reply:
I thought it looks nicer, but no specific reason, no.

Reply:
agree with tim that no placeholder is needed since we have no idea what a good default size will be.